modules/RTC/RTCUtils.js

```diff
@@ -24,6 +24,10 @@ var devices = {
     video: true
 };
 
+var audioOuputDeviceId = ''; // default device
+
+var featureDetectionVideoEl = document.createElement('video');
```

Comment:
Isn't it better to test on audio element since we are going to use the setSinkId method only on audio elements?